docs/TRIBE.md

```diff
@@ -85,4 +86,25 @@ all-nodes 	 2       			 0 		     0
 
 From this point forward, any plugins or tasks you load will load into both members of this agreement.
 
-*Note: Once the cluster is started subsequent new nodes can choose to establish membership through **any** node as there is no "master".*
\ No newline at end of file
+*Note: Once the cluster is started subsequent new nodes can choose to establish membership through **any** node as there is no "master".*
+
+### Examples
+
+#### Starting a 4 node cluster and listing members
+![tribe-connect-seed](https://user-images.githubusercontent.com/18285077/28823050-0b8e223c-76bc-11e7-88a4-fa601a03439d.gif)
+
+#### Creating an agreement and joining members to it
+![tribe-create-join-agreement](https://user-images.githubusercontent.com/18285077/28823052-0ea6b9ca-76bc-11e7-97d9-16b9b0018709.gif)
+
+#### Loading plugins and starting tasks in tribe mode
+
+Loading plugins on a node participating in an agreement:
```

Comment:
Suggestion to change that to section: `#### Loading plugins on a node participating in an agreement:`